apps/researcher/src/app/[locale]/persons/person-list.tsx

```diff
@@ -0,0 +1,24 @@
+import PersonCard from './person-card';
+import {useTranslations} from 'next-intl';
+import {SearchResult} from '@/lib/persons';
+
+interface Props {
+  persons: SearchResult['persons'];
+  totalCount: SearchResult['totalCount'];
+}
+
+export default function PersonList({persons, totalCount}: Props) {
+  const t = useTranslations('Home');
+
+  if (totalCount) {
```

Comment:
This could be confusing: is `totalCount` a boolean ('has total count') or a number? To be explicit, check for a number (`if (totalCount > 0)`)?